capella/setup_workspace.py

```diff
@@ -39,12 +67,11 @@ def replace_config(path: pathlib.Path, key: str, value: str) -> None:
         "false",
     )
 
-    eclipse_settings_base_path = pathlib.Path(
-        "/workspace/.metadata/.plugins/org.eclipse.core.runtime/.settings"
-    )
-    # Set default Git path to /workspace/git
+    # Set default EGit path to /workspace/git
     replace_config(
```

Comment:
It might make sense to also move the existing two `replace_config` function calls to a proper function that explains what and why we are doing this change (similar as you did for the `set_git_merge_mode` which I really like).
However, as this is not really part of this PR, we can also just create an issue for that (if you think it is a good idea), so we can do it in the future.